lib/src/webcrypto/webcrypto.hmac.dart

```diff
@@ -219,7 +230,7 @@ abstract class HmacSecretKey {
   /// instead, these methods computes a signature and does a
   /// fixed-time comparison.
   /// {@endtemplate}
-  Future<Uint8List> signBytes(List<int> data);
+  Future<Uint8List> signBytes(List<int> data) => _impl.signBytes(data);
```

Comment:
We should probably do:
```suggestion
  Future<Uint8List> signBytes(List<int> data) async => await _impl.signBytes(data);
```

It can create different stack traces